README.md

```diff
@@ -100,7 +102,25 @@ See [the elections documentation](elections) for further details.
 
 See the [vendoring documentation](VENDORING.md).
 
-# Vulnerability Disclosure
+# Vulnerability handling
+
+Handling vulnerabilities in Kata is handled by the Vulnerability Management Team (VMT).
+There are generally two phases:
+- The reporting of a vulnerability to the VMT
+- Handling and disclosure of the vulnerability by the VMT
+
+## Reporting Vulnerabilities
+
+Vulnerabilities in Kata should be reported using the
+[responsible disclosure](https://en.wikipedia.org/wiki/Responsible_disclosure) model.
+
+There are two methods available to report vulnerabilities to the Kata community:
+
+1) Report via a private issue on the [Kata Containers launchpad](https://launchpad.net/katacontainers.io)
```

Comment:
If I go to https://launchpad.net/katacontainers.io (without being logged into launchpad.net), I see a number of open issues. This *may* suggest to some that the bug reports are not confidential when reported (even though they are). Hence, it might be worth adding more explanation here:

- You need a lp login to report, but when you open a bug report:
- only the (small number of) members of that group get to see the report. For everyone else (logged into launchpad or not), the bug report is invisible.
- the bug will remain invisible until ... "$details".

That said, it may make more sense to augment the text on https://launchpad.net/katacontainers.io itself though?

/cc @devimc.

In fact, @devimc, the lp page states, 'When submitting an issue, please select "Private Security."' Two questions:

- What happens if you don't do this?
- Could "Private Security" be made the default?

Reply:
> That said, it may make more sense to augment the text on https://launchpad.net/katacontainers.io itself though?

what about https://launchpad.net/katacontainers.io has a link to this document? , unfortunately I was kicked out from there and only @kata-containers/architecture-committee can modify it :disappointed:

Reply:
cc @egernst